ops/internal/manage/staging.go

```diff
@@ -215,3 +222,39 @@ func StagedSuperchainDefinition(rootP string) (*config.SuperchainDefinition, err
 
 	return sM, err
 }
+
+// isChainInDependencySet checks if a chain ID is in the supplied dependency set
+func isChainInDependencySet(chainID common.Hash, depSet *prestate.DependencySet) bool {
+	if depSet == nil {
+		return false
+	}
+	for id := range depSet.Dependencies {
+		if id == chainID.Hex() {
+			return true
+		}
+	}
+	return false
+}
+
+// convertToSCRDepset converts a prestate.DependencySet to a config.DependencySet
+func convertToSCRDepset(depSet *prestate.DependencySet) map[string]config.Dependency {
+	if depSet == nil {
+		return nil
+	}
+	if depSet.Dependencies == nil {
+		return nil
+	}
+	ds := make(map[string]config.Dependency, len(depSet.Dependencies))
+	for k, v := range depSet.Dependencies {
+		decimalChainId, err := common.HexToHash(k).Big().MarshalText()
```

Comment:
Converting from hex to decimal here to be consistent with what we do elsewhere in the registry.